### PR TITLE
 Make JWT claims reachable from SecurityContext when used OAuth2 + JWT

### PR DESCRIPTION
**Pull request by rifatdover**
- Please make sure the below checklist is followed for Pull Requests.

- [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [x] Tests are added where necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

I messed PR #7326 reopening a PR

### CODE REVIEW

File: generators/server/templates/src/main/java/package/config/oauth2/OAuth2JwtAccessTokenConverter.java.ejs [hunk 1]
```diff
@@ -95,4 +96,36 @@ public class OAuth2JwtAccessTokenConverter extends JwtAccessTokenConverter {
...
+     *      if(details instanceof OAuth2AuthenticationDetails) {
+     *          Object decodedDetails = ((OAuth2AuthenticationDetails) details).getDecodedDetails();
+     *          if(decodedDetails != null &amp;&amp; decodedDetails instanceof Map)
+     *          {
+     *             String detailFoo = ((Map) decodedDetails).get("foo");
+     *          }
+     *      }
+     *  }
+
+     * </code>
```
Comment (mraible) [inline generators/server/templates/src/main/java/package/config/oauth2/OAuth2JwtAccessTokenConverter.java.ejs]: Please change this example to fit our Java standards - curly brace on same line, space after if should do it. Thanks!

**Comment by jdubois** [comment]
Trusting @mraible on this, thanks a lot @rifatdover !

**Comment by AlexandreCassagne** [comment]
Has this been added to 5.0.0-beta.1?

**Comment by pascalgrimaud** [comment]
@AlexandreCassagne : yes

**Pull request merged**